docs/releases/v2023.1.rst

```diff
@@ -0,0 +1,120 @@
+Gluon 2023.1
+============
+
+Important notes
+---------------
+
+Upgrades to v2023.1 and later releases are only supported from releases v2021.1 and later.
+This is due to migrations that have been removed to simplify maintenance.
+
+
+Added hardware support
+----------------------
+
+ath79-generic
+~~~~~~~~~~~~~
+
+- Extreme Networks
+
+  - WS-AP3805i
+
+ath79-nand
+~~~~~~~~~~
+
+- GL.iNet
+
+  - GL-XE300
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - EAP615-Wall
+
+- Wavlink
+
+  - WS-WN572HP3 4G
+
+ramips-mt76x8
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - TL-MR6400 (v5)
+
+
+Features
+--------
+
+DNS Caching
+~~~~~~~~~~~
+
+DNS-Caching using the dnsmasq resolver is reintroduced.
+
+See the :ref:`DNS-Caching documentation <dns-caching>` section for
+details on how to enable this feature.
+
+Cellular Modem Support
+~~~~~~~~~~~~~~~~~~~~~~
+
+Support for using cellular data connections as the primary uplink connection has been added
+to Gluon. This is supported for hardware that comes with a built-in cellular modem. The required user
+configuration for the APN and SIM-PIN can be performed in the Advanced Settings in Config Mode.
+
+Interface Role UI
+~~~~~~~~~~~~~~~~~
+
+Interface Roles can now be assigned from the Network page of the Advanced Settings
+in Config Mode. This configuration is preserved on Gluon upgrades.
+
+WireGuard Key Translation
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This release adds a new mechanism for seamlessly translating existing fastd private keys on the nodes into
+WireGuard pendants whose public keys can be **separately** translated on the servers.
+This mitigates the need to re-exchange public-keys for communities when migrating to WireGuard-based VPN.
+
+See the :ref:`gluon-mesh-vpn-key-translate <gluon-mesh-vpn-key-translate>` section for details.
+
+
+Bugfixes
+--------
+
+- Custom channel lists using a radios ``channels`` UCI option are now preserved on upgrade
+  in case ``gluon.wireless.preserve_channels`` is set.
+- Custom HT modes for radios are now preserved when ``gluon.wireless.preserve_channels``
+  is set.
+- Broken mesh links between MediaTek 11ax and Qualcomm 11ac hardware are worked around.
```

Comment:
Do we have an issue number or commit we can link? I have no idea what this point means.

Reply:
https://github.com/freifunk-gluon/gluon/commit/1f09b1594c1eb619aad443e172445df98f2cc137 is the commit
the pr was: https://github.com/freifunk-gluon/gluon/pull/2905